src/me/mrCookieSlime/Slimefun/listeners/ItemListener.java

```diff
@@ -378,12 +378,16 @@ else if (e.getEntity() instanceof Wither) {
 	@EventHandler
     public void onAnvil(InventoryClickEvent e) {
         if (e.getRawSlot() == 2 && e.getWhoClicked() instanceof Player && e.getInventory().getType() == InventoryType.ANVIL) {
-		if (SlimefunManager.isItemSimiliar(e.getInventory().getContents()[0], SlimefunItems.ELYTRA, true)) return;
-
-        	if (SlimefunItem.getByItem(e.getInventory().getContents()[0]) != null && !SlimefunItem.isDisabled(e.getInventory().getContents()[0])) {
-            	e.setCancelled(true);
-                Messages.local.sendTranslation((Player) e.getWhoClicked(), "anvil.not-working", true);
-            }
+			if (SlimefunManager.isItemSimiliar(e.getInventory().getContents()[0], SlimefunItems.ELYTRA, true)) return;
+			
+			/* SlimefunItems.BROKEN_SPAWNER 
+			 * does SlimefunItem.isDisabled not apply to this item for some reason?
```

Comment:
Probably because getByItem has no special case for Broken Spawners and tries to compare the lore?